### PR TITLE
Add the dracula.nvim plugin

### PR DESCRIPTION
**Pull request by refaelsh**
This PR adds the dracula.nvim plugin.
Here the link to nixpkgs: https://search.nixos.org/packages?channel=24.05&show=vimPlugins.dracula-nvim&from=0&size=50&sort=relevance&type=packages&query=dracula-nvim.

Thanks in advance for reviewing :-)

### CODE REVIEW

File: plugins/colorschemes/dracula-nvim.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+lib.nixvim.neovim-plugin.mkNeovimPlugin config {
```
Comment (MattSturgeon) [inline plugins/colorschemes/dracula-nvim.nix:7]: This recently changed, 

```suggestion
lib.nixvim.neovim-plugin.mkNeovimPlugin {
```

File: plugins/colorschemes/dracula-nvim.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  name = "dracula";
+  originalName = "dracula.nvim ";
+  defaultPackage = pkgs.vimPlugins.dracula-nvim;
```
Comment (MattSturgeon) [inline plugins/colorschemes/dracula-nvim.nix:10]: This is currently correct, but may change if #2139 is merged first.

File: plugins/colorschemes/dracula-nvim.nix [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  lib,
+  config,
```
Comment (MattSturgeon) [inline plugins/colorschemes/dracula-nvim.nix:3]:
```suggestion
```

**Review by MattSturgeon — CHANGES_REQUESTED** [review]
CI failure is caused by #2140

Otherwise everything looks good.

If you could rebase so we no longer have the "merge" commit, that'd be great. Otherwise we'll tackle squashing/rebasing.

**Comment by refaelsh** [comment]
I did something bad. Not really sure. Probably the "rebasing" failed. Sorry for the inconvinience. I will open a new PR.

**Comment by MattSturgeon** [comment]
@refaelsh did you close this intentionally? I didn't think there was much work needed to get it finished and merged 😨

**Comment by refaelsh** [comment]
> @refaelsh did you close this intentionally? I didn't think there was much work needed to get it finished and merged 😨

I did something bad. Not really sure. Probably the "rebasing" failed. Sorry for the inconvinience. I will open a new PR.

**Comment by MattSturgeon** [comment]
Ah, perhaps you did something like `git push --delete origin add-dracula-nvim` or `git push origin :add-dracula-nvim` (equivalent).

I think github usually has a "restore branch" button on PRs, once restored it should be possible to re-open.

**Comment by refaelsh** [comment]
Here is the new PR: https://github.com/nix-community/nixvim/pull/2153.

**Pull request closed without merging**